anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -168,17 +219,29 @@ def __init__(self, metadata=None):
                 raise Exception(
                     "Must have feed metadata in db already, should sync metadata before invoking instance operations"
                 )
-
+        self._log_ctx: LogContext = LogContext(None, self.__feed_name__, None)
```

Comment:
Should this be _log_context to match the getter and setter below?

Reply:
The name of the property cannot collide with the variable name, so I had to name it something slightly different.